src/Makefile

```diff
@@ -42,6 +42,7 @@ OBJS = ConstantDefineMod.o \
        SnowLayerDivideMod.o \
        SnowLayerWaterComboMod.o \
        SnowpackCompactionMod.o \
+       SnowpackCompactionAR2023Mod.o \
```

Comment:
should also include the Dependencies in the below of this file, e.g.
SnowpackCompactionMod.o:              ../utility/Machine.o NoahmpVarType.o ConstantDefineMod.o

Reply:
this should be 2024 not 2023